### PR TITLE
Fix const redeclaration error

### PR DESCRIPTION
**Pull request by lpinca**
This fixes a bug introduced in 2f9b6a43adc8ba647ba77c5809d062b3d55aba0b.

### CODE REVIEW

**Comment by ghost** [comment]
[Travis build passed](https://travis-ci.org/nodejs/nodejs.org/builds/118459250) :+1:

File: events/yaml-sync.js [hunk 1]
```diff
@@ -8,10 +8,11 @@ var yaml = require('js-yaml'),
   store = yaml.safeLoad(str)
 
 exports.getRegion = (region) => {
-  for (const reg in store.regions) {
+  let reg;
+  for (reg in store.regions) {
     if (store.regions[reg].region === region) return store.regions[reg]
   }
-  const reg = { region: region }
```
Comment (lpinca) [inline events/yaml-sync.js:14]: This works only in strict mode, which is currently not enabled.

**Comment by hiroppy** [comment]
Oh, I'm sorry. Thank you:)

**Comment by lpinca** [comment]
No problem, my bad, I didn't see it when reviewing.

**Pull request merged**